tests/vmi_networking_test.go

```diff
@@ -722,9 +722,14 @@ var _ = Describe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][le
 				ipAddr := gatewayIPFromCIDR(networkCIDR)
 				Expect(libnet.PingFromVMConsole(serverVMI, ipAddr)).To(Succeed())
 
-				By("Checking ping (IPv4) to google")
-				Expect(libnet.PingFromVMConsole(serverVMI, "8.8.8.8")).To(Succeed())
-				Expect(libnet.PingFromVMConsole(clientVMI, "google.com")).To(Succeed())
+				By("Checking ping (IPv4)")
+				if len(flags.IPV4ConnectivityCheckAddress) != 0 {
+					Expect(libnet.PingFromVMConsole(serverVMI, flags.IPV4ConnectivityCheckAddress)).To(Succeed())
+				}
```

Comment:
perhaps it would be informative to add a message to notify the tester that the ping ipv4 connectivity test was skipped?

Reply:
Done.